raillabel_providerkit/validation/issue.py

```diff
@@ -79,45 +85,20 @@ def deserialize(cls, serialized_issue_identifiers: dict[str, str | int]) -> "Iss
         TypeError
```

Comment:
The TypeError doesn't seem to be raised anymore, so this could be removed from the docstring